js/script.js

```diff
@@ -0,0 +1,21 @@
+$(function () { // Same as document.addEventListener("DOMContentLoaded"...
+  $.get("http://api.openweathermap.org/data/2.5/weather?q=Pretoria,za&units=metric&APPID=ad072cb9c0700a66a327e65573d6cee8", function(response) {
+    var weatherresult = response;
+    var name = "City: " + weatherresult.name;
+    var temp = "Temperature: " + weatherresult.main.temp + "°C";
+    var tempMax = "Maxmimum Temperature: " + weatherresult.main.temp_max + "°C";
+    var tempMin = "Minimum Temperature: " + weatherresult.main.temp_min + "°C";
+    var pressure = "Pressure: " + weatherresult.main.pressure + " Pa";
+    var humidity = "Humidity: " + weatherresult.main.humidity + " g/m3";
+    var weatherdescription = "Weather description: " + weatherresult.weather[0].main;
```

Comment:
What happens if weatherresult.weather array is empty?

Reply:
.... only knows